lib/CustomFields/components/CustomFieldsForm/CustomFieldsForm.js

```diff
@@ -230,10 +235,13 @@ const CustomFieldsForm = ({
   );
 
   const getFieldTypesOptions = () => {
-    return Object.values(fieldTypes).map(fieldType => ({
-      label: fieldTypesLabels[fieldType],
-      value: fieldType,
-    }));
+    return Object.values(fieldTypes)
+      .map(fieldType => ({
+        labelElement: fieldTypesLabels[fieldType],
+        labelText: intl.formatMessage({ id: fieldTypesLabelIds[fieldType] }),
```

Comment:
Could we use `label` for sorting and display it in the button to not introduce separate variables like `labelElement` and `labelText`?.